inst/include/testthat/testthat.h

```diff
@@ -123,7 +123,8 @@ inline Catch::Session& catchSession()
 
 inline bool run_tests()
 {
-  return catchSession().run() == 0;
+  const char* argv[] = {"catch", "-r", "xml"};
+  return catchSession().run(3, argv) == 0;
```

Comment:
Surely there must be a better way to do this, but the docs related to this only seem to reference the command line interface (https://github.com/catchorg/Catch2/blob/master/docs/command-line.md#choosing-a-reporter-to-use)

Reply:
To the best of my knowledge this is indeed the only way to go :(